### DISCUSSION TITLE
GPT 4o support?

### DISCUSSION THREAD

**Discussion by Unlok-Simon** [janhq/jan#2918] [General]
Any thoughts on when we will be able to select GPT 4o as the model for the OpenAI API? Keen to be able to use it from Jan for performance and cost reasons.

**Comment by khandar-william** [comment]
Since I saw the configuration exists: https://github.com/janhq/jan/blob/dev/extensions/inference-openai-extension/resources/models.json
Currently I add it manually:
1. in `/jan/models` create new directory `gpt-4o-custom` (or any name)
2. create one file `model.json`
3. fill `model.json` with 
```json
{
    "sources": [
      {
        "url": "https://openai.com"
      }
    ],
    "id": "gpt-4o",
    "object": "model",
    "name": "OpenAI GPT 4o",
    "version": "1.1",
    "description": "OpenAI GPT 4o is a new flagship model with fast speed and high quality",
    "format": "api",
    "settings": {},
    "parameters": {
      "max_tokens": 4096,
      "temperature": 0.7,
      "top_p": 0.95,
      "stream": true,
      "stop": [],
      "frequency_penalty": 0,
      "presence_penalty": 0
    },
    "metadata": {
      "author": "OpenAI",
      "tags": [
        "General"
      ]
    },
    "engine": "openai"
  }
```
(taken from [here](https://github.com/janhq/jan/blob/dev/extensions/inference-openai-extension/resources/models.json))

**Comment by Unlok-Simon** [comment]
Saw this has been added in the latest release 0.4.13. Thanks!
